### DISCUSSION TITLE
Does it support macOS ?

### DISCUSSION THREAD

**Discussion by SENPAY98K** [dyc3/steamguard-cli#341] [Q&A] [answered]
Is there a way to run it in macOS ?

**Comment by dyc3** [comment, marked as answer]
Yes, but you have to build it yourself. I don't have a mac so I can't publish mac builds.

The easiest way to do that is to download and install [Rust](https://www.rust-lang.org/), and run `cargo install steamguard-cli` in your terminal.

**Reply by hrxn** [reply]
GitHub Actions to build automagically for mac?

**Comment by SENPAY98K** [comment]
Okay i will try it
